exporter/elasticsearchexporter/README.md

```diff
@@ -58,6 +58,7 @@ This exporter supports sending OpenTelemetry logs and traces to [Elasticsearch](
   - `max_requests` (default=3): Number of HTTP request retries.
   - `initial_interval` (default=100ms): Initial waiting time if a HTTP request failed.
   - `max_interval` (default=1m): Max waiting time if a HTTP request failed.
+  - `retry_on_status` (default=[429, 500, 502, 503, 504]): Status codes that trigger request or document level retries. Request level retry and document level retry status codes are shared and cannot be configured separately. To avoid duplicates, it is recommended to set it to `[429]`, which would be the default in a future version.
```

Comment:
Could we add a notice or warning here saying that the default will be changed to `[429]` in the future.  Mentioning that in the changelog is good but I think it needs to be more readily visible to users as well.

Reply:
done!